client/app/components/Parameters.jsx

```diff
@@ -137,6 +143,8 @@ export class Parameters extends React.Component {
 
   renderParameter(param, index) {
     const { editable } = this.props;
+    const isEmpty = param.pendingValue === '' || param.pendingValue === null;
```

Comment:
Some notes:
- After #4049 the `pendingValue` should be normalized as well and empty values will be standardized as null :rocket:
- Should we check if the Parameter Value is empty in case there is no pendingValue? (e.g: https://deploy-preview-4197--redash-preview.netlify.com/queries/216 has empty values)

Reply:
This line of code is totally temporary.
@gabrieldutra wdyt of this for `Parameter.js?`
```js
get isEmpty() {
  const value = this.hasPendingValue ? this.pendingValue : this.value;
  return this.isEmptyValue(value);
}
```
It makes sense to me that `param.isEmpty` always responds with the currently selected value.

Reply:
Currently the most relevant usage of that is to determine the parameters with missing values prior to a query execution, so that would need to be updated (in this case it's necessary to consider the parameter value, not the pending value).

I'm actually "centered positioned" on that. Depends if it makes more sense for the Parameter structure to reflect the "actual Input" or the "Query Execution". For the first one the pendingValue is part of it, for the second one it's just a "state" the parameter has. Considering the existence of `this.isEmptyValue(anyValue)` it doesn't matter the direction we choose.

BTW, considering "Can also add "Invalid input" error (i.e. NaN in InputNumber)", one idea is to remove the `isEmpty` (no usage besides deciding if there is an error prior to the query execution) and work on a generalized `getErrors`.

Reply:
I'm for `param.getCurrentInputError()` which would return a string ("Required field", "Invalid input", etc) or null. I can add it in this PR 👍